media-plugins/dragonfly-reverb/dragonfly-reverb-9999.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A set of free reverb effects"
+HOMEPAGE="https://github.com/michaelwillis/dragonfly-reverb"
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3 autotools
+	EGIT_REPO_URI="https://github.com/michaelwillis/dragonfly-reverb.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/michaelwillis/dragonfly-reverb/releases/download/${PV}/DragonflyReverb-Source-v${PV}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/DragonflyReverb-Source-v${PV}"
+fi
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="jack"
+
+RDEPEND="x11-libs/libX11
+virtual/opengl
+jack? ( virtual/jack )
+"
+
+DOCS=( README.md )
+
+src_prepare() {
+	default
+
+	# Remove automagic deps
+	sed -i -re '/^HAVE_(CAIRO|OPENGL|X11|JACK|LIBLO)/d' dpf/Makefile.base.mk || die 'sed failed'
+	# Do not strip
+	sed -i -re 's/-Wl,--strip-all//g' common/Makefile.mk
```

Comment:
Is this still necessary? I don't see any `strip` commands in this Makefile (https://github.com/michaelwillis/dragonfly-reverb/blob/3.2.1/common/Makefile)?

Reply:
It's not a command, it's a linker option. But you're right, this isn't necessary: this Makefile is only used for the `common` subdirectory, which only builds object files, no shared libs itself, so it never actually runs the linker. `SKIP_STRIPPING=true` below is necessary and sufficient to avoid stripping the final binaries.